kasa/smartcam/modules/device.py

```diff
@@ -26,6 +33,32 @@ def _initialize_features(self) -> None:
                 type=Feature.Type.Sensor,
             )
         )
+        if self.rssi is not None:
+            self._add_feature(
+                Feature(
+                    self._device,
+                    container=self,
+                    id="rssi",
+                    name="RSSI",
+                    attribute_getter="rssi",
+                    icon="mdi:signal",
+                    unit_getter=lambda: "dBm",
+                    category=Feature.Category.Debug,
+                    type=Feature.Type.Sensor,
+                )
+            )
+            self._add_feature(
+                Feature(
+                    self._device,
+                    container=self,
+                    id="signal_level",
```

Comment:
should this have its own availability check? The PR looks good, but I can still give it a try on my cam soon.

The only potential concern would be that, given this being a part of the initial query, not every camera supports this, but it's probably unlikely.